indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1641,136 +1596,41 @@ def getBody(self, story=None, indexedFlowable={}, level=1 ):
-                    else:
-                        lp.append(Paragraph("", styleRegistrant))
-                elif key == "ArrivalDate":
-                    if reg.getAccommodation() is not None and reg.getAccommodation().getArrivalDate() is not None:
-                        lp.append(Paragraph("%s"%escape(reg.getAccommodation().getArrivalDate().strftime("%d-%B-%Y")), styleRegistrant))
-                    else:
-                        lp.append(Paragraph("", styleRegistrant))
-                elif key == "DepartureDate":
-                    if reg.getAccommodation() is not None and reg.getAccommodation().getDepartureDate() is not None:
-                        lp.append(Paragraph("%s"%escape(reg.getAccommodation().getDepartureDate().strftime("%d-%B-%Y")), styleRegistrant))
-                    else:
-                        lp.append(Paragraph("", styleRegistrant))
-                elif key == "ReasonParticipation":
-                    lp.append(Paragraph("""%s"""%escape(reg.getReasonParticipation() or ""), styleRegistrant))
-                elif key == "RegistrationDate":
-                    if reg.getRegistrationDate() is not None:
-                        lp.append(Paragraph("""%s"""%escape(reg.getAdjustedRegistrationDate().strftime("%d-%B-%Y-%H:%M")), styleRegistrant))
-                    else:
-                        lp.append(Paragraph("", styleRegistrant))
-                elif key.startswith("s-"):
-                    ids=key.split("-")
-                    if len(ids)==2:
-                        status=reg.getStatusById(ids[1])
-                        cap=""
-                        if status.getStatusValue() is not None:
-                            cap=status.getStatusValue().getCaption()
-                        lp.append(Paragraph("""%s"""%escape(cap), styleRegistrant))
-                else:
-                    ids=key.split("-")
-                    if len(ids)==2:
-                        group=reg.getMiscellaneousGroupById(ids[0])
-                        if group is not None:
-                            i=group.getResponseItemById(ids[1])
-                            if i is not None:
-                                lp.append(Paragraph("""%s"""%escape(str(i.getValue())), styleRegistrant))
-                                continue
-                    lp.append(Paragraph("", styleRegistrant))
+            lp.append(Paragraph("{} {}".format(registration.first_name.encode('utf-8'),
+                                               registration.last_name.encode('utf-8')), text_format))
+            data = registration.data_by_field
+            for item in self._display:
+                value = data[item.id].friendly_data if item.id in data else ''
+                if isinstance(value, unicode):
+                    value = value.encode('utf-8')
+                lp.append(Paragraph("{}".format(value), text_format))
```

Comment:
just use `str()` instead of a format string that consists of nothing but a single placeholder...